### PR TITLE
change line to read the payload correctly

### PR DESCRIPTION
**Pull request by lee-leonardo**

### CODE REVIEW

File: src/controllers/manifests.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ exports.create = function (client, storage, pwabuilder, raygun) {
           .fail(function (err) {
             return res.status(422).json({ error: err.message });
           });
-      } else if (req.body.manifest) {
+      } else if (req.body.content && req.body.format === "w3c") {
         pwabuilder
           .uploadManifest(req.body.manifest, client)
```
Comment (JudahGabriel) [inline src/controllers/manifests.js:38]: You'd need to update this too, right? Still looking at req.body.manifest

**Review by JudahGabriel — CHANGES_REQUESTED** [review]

File: src/services/pwabuilder.js [hunk 2]
```diff
@@ -36,17 +36,21 @@ PWABuilder.prototype.createManifestFromUrl = function (url, client) {
...
+                manifest.id,
+                JSON.stringify(manifest),
+                'EX',
+                expirationTime
+              );
+              return resolve(manifest);
+            }).fail(reject);
+        } catch (e) {
+          return reject(e);
+        }
```
Comment (lee-leonardo) [inline src/services/pwabuilder.js:53]: exception kills the service, so this mitigates the effects of us calling it twice

File: src/controllers/manifests.js [hunk 3]
```diff
@@ -33,9 +33,9 @@ exports.create = function (client, storage, pwabuilder, raygun) {
           .fail(function (err) {
             return res.status(422).json({ error: err.message });
           });
-      } else if (req.body.manifest) {
+      } else if (req.body.content && req.body.format === "w3c") {
         pwabuilder
-          .uploadManifest(req.body.manifest, client)
+          .uploadManifest(req.body, client)
```
Comment (JudahGabriel) [inline src/controllers/manifests.js:38]: Do we need to update `uploadManifest` to handle the manifest properly? The UI is sending in a ManifestDetectionResult object, which contains a `.content` property with the actual manifest. Does `uploadManifest` need to handle that?

Comment (lee-leonardo) [reply src/controllers/manifests.js:38]: And to respond to your comment, the format saved in redis is the format that has the actual manifest is the one we're passing e.g.

{
  content: ManifestDetectionResult,
  default...
  ad naseum...
  id: "crypto has uuid etc"
}

**Review by JudahGabriel — APPROVED** [review]

**Pull request merged**